concourse/setup.py

```diff
@@ -98,30 +98,33 @@ def create_image_pull_secret(
 
 
 @ensure_annotations
-def create_tls_secret(namespace: str, config_path: str):
-    """Creates the TLS secret for concourse web component in the K8s cluster"""
-    # XXX: move those to cc-config
-    tls_key_file = "concourse-tls.key"
-    tls_cert_file = "concourse-tls.crt"
-    tls_secret_name = "concourse-web-tls"
-
-    secret_helper = KubernetesSecretHelper(kubeutil.Ctx().create_core_api())
+def create_tls_secret(
+    config_set: ConfigurationSet,
```

Comment:
accept TlsSecret as input; while at it: introduce a dedicated model element type TlsConfig